rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -4246,4 +4252,33 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
             )
         }
     }
+
+    String createMisfireReport(ScheduledExecution scheduledExecution, Date misfireTime) {
```

Comment:
@gschueler should the report be a string or structured data that is rendered at view time? It seems the ExecReport message indicates a string, but if you have a better way I'll see if I can make it work.